packages/gatsby-transformer-mdx-introspection/README.md

```diff
@@ -40,31 +47,122 @@ query MyQuery {
 }
 ```
 
-Please not the lowercase `apitype` in the query - the names are normalized to lowercase by the DOM parser used.
-
-Example response:
+### Example response:
 
 ```json
 {
   "data": {
     "allComponentInMdx": {
       "nodes": [
         {
+          "component": "ApiType",
           "attributes": [
             {
-              "name": "apikey",
+              "name": "apiKey",
               "value": "test"
             },
             {
               "name": "type",
               "value": "OutOfOrderPropertiesTestType"
             }
           ],
-          "page": {
-            "fields": {
-              "slug": "/api/types/"
+          "mdx": {
+            "file": {
+              "relativePath": "api/types/index.md"
             }
           }
         },
-...
+        ...
+      ]
+    }
+  }
+}
+```
+
+## Options
+
+**`lowercaseIdentifiers`** [boolean] (optional, defaults to false)
```

Comment:
I suggest to drop that legacy support.  This plugin is in a pre-1.0 alpha release phase and the lowercasing was never actually a wanted feature.

Reply:
I'm fine with dropping support for this; as you said it was intended entirely as an option to preserve legacy behavior.